src/test/java/edu/ucsb/cs156/courses/documents/CoursePageTests.java

```diff
@@ -0,0 +1,58 @@
+package edu.ucsb.cs156.courses.documents;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class CoursePageTests {
```

Comment:
Can we add explicit access modifiers to the class declaration and the methods in this file?

Reply:
fixed